tests/contrib/httpx/test_httpx.py

```diff
@@ -0,0 +1,207 @@
+import contextlib
+import os
+
+import httpx
+import pytest
+
+from ddtrace import config
+from ddtrace.contrib.httpx.patch import patch
+from ddtrace.contrib.httpx.patch import unpatch
+from ddtrace.settings.http import HttpConfig
+from tests.utils import override_config
+from tests.utils import override_http_config
+from tests.utils import snapshot_context
+
+
+# host:port of httpbin_local container
+HOST = "localhost"
+PORT = 8001
+
+
+def get_url(path):
+    # type: (str) -> str
+    return "http://{}:{}{}".format(HOST, PORT, path)
+
+
+@pytest.fixture(autouse=True)
+def patch_httpx():
+    patch()
+    try:
+        yield
+    finally:
+        unpatch()
+
+
+@contextlib.contextmanager
+def _snapshot_context(request):
```

Comment:
Can we move this helper to conftest? It'd be useful in general across our integrations

Reply:
Done in #2722